src/rest_framework_jwt/management/commands/obtain_token.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+from django.conf import settings
+from rest_framework_jwt.settings import api_settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Create new jwt token for the requested user during development'
+
+    def add_arguments(self, parser):
+        parser.add_argument(dest='pk', type=str, help="User's primary key")
+
+    def handle(self, pk, *args, **options):
```

Comment:
Is it necessary to explicitly specify the `pk` parameter?

Reply:
depend on what is the definition of necessary, I defined pk explicitly since **handle()** will expect to receive a pk parameter anyway and **Explicit is better than implicit.** according to python zen. 
if you think it could have any downside, I would be glad to hear

Reply:
Whatever works for you.